graphene/types/schema.py

```diff
@@ -164,8 +174,7 @@ def create_scalar(graphene_type):
             parse_literal=getattr(graphene_type, "parse_literal", None),
```

Comment:
So apparently I can't leave a comment on the line I actually want to leave the comment so I'll put it here... 
Shouldn't the `create_scalar` method above also get the same modif of the name?

Reply:
Absolutely, thanks for the catch. Fixed